src/script/conversation/ConversationService.ts

```diff
@@ -254,10 +254,7 @@ export class ConversationService {
    * @param userId ID of member to be removed from the the conversation
    * @returns Resolves with the server response
    */
-  deleteMembers(
-    conversationId: string | QualifiedId,
-    userId: string | QualifiedId,
-  ): Promise<ConversationMemberLeaveEvent> {
+  deleteMembers(conversationId: QualifiedId, userId: QualifiedId): Promise<ConversationMemberLeaveEvent> {
```

Comment:
should we consider backwards compatability?

Reply:
discussed this with @aweiss-dev, the only one place its being used in already passes qualifiedid type

Reply:
ok nice